test/parallel/test-inspector-multisession-js.js

```diff
@@ -61,3 +62,44 @@ test().then(() => {
   clearInterval(interval);
   console.log('Done!');
 });
+
+async function promiseTest() {
+  const session1 = new PromiseSession();
+  const session2 = new PromiseSession();
+
+  session1.connect();
+  session2.connect();
+
+  let session1Paused = false;
+  let session2Paused = false;
+
+  session1.on('Debugger.paused', () => session1Paused = true);
+  session2.on('Debugger.paused', () => session2Paused = true);
+
+  console.log('Connected');
```

Comment:
If the `console.log()` messages are not part of the test, I'd prefer them as comments instead. If they are part of the test, please add a comment indicating so.